api/v1beta1/agentserviceconfig_types.go

```diff
@@ -88,6 +88,12 @@ type AgentServiceConfigSpec struct {
 	// that are used if one the operators fails to be successfully deployed
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Must-Gather Images"
 	MustGatherImages []MustGatherImage `json:"mustGatherImages,omitempty"`
+
+	// ExposeIPXEHTTPRoute is controlling whether the operator is creating plain HTTP routes
+	// iPXE hosts may not work with router cyphers and may access artifacts via HTTP only
+	// This setting defaults to false, accepts "true,false"
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Expose IPXE HTTP route"
+	ExposeIPXEHTTPRoute string `json:"exposeIPXEHTTPRoute,omitempty"`
```

Comment:
Moving the "expose" part into the value will give more flexibility and fit the intent of keeping this as a string. For example, call the field `ipxeHTTPRoute` with values `enabled` and `disabled`. That leaves the door open for some other mode later, like perhaps `authenticated` or `ipfiltered` or whatever else we might make up.

Reply:
Reworked to use `enabled`/`disabled`